chain/network-primitives/src/lib.rs

```diff
@@ -1 +1,2 @@
 pub mod types;
+pub mod utils;
```

Comment:
please also add the README.md file to network-primitives to describe what kind of functions should end up in this crate